source/includes/connection-snippets/scram-snippets/c-connection.c

```diff
@@ -0,0 +1,24 @@
+// begin scram connection
+#include <mongoc/mongoc.h>
+
+int
+main (int argc, char *argv[])
+{
+   mongoc_database_t *database;
+   mongoc_client_t *client;
+
+   mongoc_init ();
+
+   client = mongoc_client_new(
```

Comment:
The spacing for function calls seems inconsistent. This line does not include a space before the opening parens, but 5, 15, 17, 18, and 20 do. Should this one include a space too?